trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesDatabaseService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Tripal Cultivate Phenotypes Database service definition.
+ */
+
+namespace Drupal\trpcultivate_phenotypes\Service;
+
+/**
+ * Class TripalCultivatePhenotypesDatabaseService.
+ */
+class TripalCultivatePhenotypesDatabaseService {
+  
+  /**
+   * Chado Db.
+   */
+  protected $chado;
+
+  /**
+   * Constructor.
+   */
+  public function __construct() {
```

Comment:
Based on the change to the services.yml, the database connection is now available here:
```suggestion
  public function __construct(ChadoConnection $connection) {
```